tests/testthat/test-model-compile.R

```diff
@@ -37,13 +37,13 @@ test_that("compile() method forces recompilation if changes in flags", {
   skip_on_cran()
   skip_on_ci() # FIXME: this test seems to pass locally but fail on travis
   expect_message(
-    mod$compile(opencl=TRUE),
+    mod$compile(threads=TRUE),
```

Comment:
opencl isnt present on Windows by default. TBB is so I think testing this behavior with threads is better. I also think that is why Travis was failing, so `skip_on_ci()` might not be needed here.

Reply:
Ok cool. I just pushed a commit removing `skip_on_ci()` so we'll see if travis can handle it!